src/infrastructure/validation/logger/ModuleID.java

```diff
@@ -1,16 +1,24 @@
-/**
- * 
- */
 package infrastructure.validation.logger;
 
 /**
+ * The ILogger interface depends on the ModuleID enum
+ * The ModuleID enum helps specify which module is calling the log
+ * Currently supported module names are:
+ *   
```

Comment:
There is a break from mentioning the module names and the actual place where it is explained. Just refer to the explanation below. You have opened a ':' in the file comments but nothing comes after it

Reply:
Ah, leftovers that escaped my previous commit :(